site/_includes/layouts/namespace-reference.njk

```diff
@@ -79,21 +79,9 @@
                     <code>{{ permission }}</code><br />
                   {% endfor %}
                   {{ extra_permissions_html | safe }}
-                </div>
-              </li>
-              <li>
-                <div class="code-sections__label">Has warning?</div>
-                <div>    
                   {{ has_warning | safe }}
```

Comment:
The current implementation of `extra_permissions_html` and `has_warning` implicitly requires that page authors manually handle line breaks between these sections. It would be better to treat each block of HTML as as completely independent in order to minimize the chance of rendering bugs. In this case, we should conditionally insert a `<br>` after `extra_permissions_html` if `has_warning` is not empty.